tajo-storage/tajo-storage-kafka/pom.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should add tajo-storage-kafk module to tajo-storage and add module copy-script in prepare-package